resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ClientDistributedCacheManager.scala

```diff
@@ -117,48 +117,62 @@ private[spark] class ClientDistributedCacheManager() extends Logging {
    * Returns a boolean to denote whether a cache file is visible to all (public)
    * @return true if the path in the uri is visible to all, false otherwise
    */
-  private def isPublic(conf: Configuration, uri: URI, statCache: Map[URI, FileStatus]): Boolean = {
-    val fs = FileSystem.get(uri, conf)
```

Comment:
@mridulm  will this cause any issues? I think it will not given we are doing the same in L71 with the first time we add to the `statCache`.

Reply:
Let us remove this change (`conf` with `fs`), it is unrelated to what we are trying to fix.
The actual call itself is cheap, since it is locally cached already.